tools/trap/ramdumpParser.py

```diff
@@ -1000,12 +998,11 @@ def main():
 	programCounter = 0
 	stackSize = 0
 	have_ram_kernel_text = False
-	gdb_path='/usr/bin/gdb'
 	nm_path='/usr/bin/nm'
 	readelf_path='/usr/bin/readelf'
```

Comment:
We need to use the tools from the docker cross toolchain. 
Please change the names as arm-none-eabi-xxxxx.
Also, we dont have to specify full path for the tools. The toolchain would already be included in the system PATH.

Reply:
When cannot execute direct toolchain command from the docker container, it does not work.